### PR TITLE
Fixes #300

### LINKED ISSUE #300: [3.0.0-beta] Error when baking template
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* BootstrapUI Version: 3.0.0-beta
* Platform and Target: Mac 10.14.6. PHP 7.3.13

### What you did
Ran `bin/cake bake template Users --prefix SuperAdmin -t BootstrapUI`

### Expected Behavior
Templates in the SuperAdmin/Users folder to be baked

### Actual Behavior
```
Exception: Unexpected token "name" of value "if" ("end of statement block" expected).
In [/project/vendor/friendsofcake/bootstrap-ui/templates/bake/Template/index.twig, line 34]

2020-02-06 05:23:22 Error: [Twig\Error\SyntaxError] Unexpected token "name" of value "if" ("end of statement block" expected). in /project/vendor/friendsofcake/bootstrap-ui/templates/bake/Template/index.twig on line 34
Stack Trace:
- /project/vendor/twig/twig/src/TokenParser/ForTokenParser.php:39
- /project/vendor/twig/twig/src/Parser.php:182
- /project/vendor/twig/twig/src/TokenParser/IfTokenParser.php:39
- /project/vendor/twig/twig/src/Parser.php:182
- /project/vendor/twig/twig/src/TokenParser/ForTokenParser.php:40
- /project/vendor/twig/twig/src/Parser.php:182
- /project/vendor/twig/twig/src/Parser.php:95
- /project/vendor/twig/twig/src/Environment.php:479
- /project/vendor/twig/twig/src/Environment.php:507
- /project/vendor/twig/twig/src/Environment.php:348
- /project/vendor/twig/twig/src/Environment.php:309
- /project/vendor/wyrihaximus/twig-view/src/View/TwigView.php:192
- /project/vendor/cakephp/bake/src/View/BakeView.php:114
- /project/vendor/cakephp/bake/src/Utility/TemplateRenderer.php:101
- /project/vendor/cakephp/bake/src/Command/TemplateCommand.php:382
- /project/vendor/cakephp/bake/src/Command/TemplateCommand.php:135
- /project/vendor/cakephp/cakephp/src/Console/BaseCommand.php:175
- /project/vendor/cakephp/cakephp/src/Console/CommandRunner.php:336
- /project/vendor/cakephp/cakephp/src/Console/CommandRunner.php:171
- /project/bin/cake.php:12
```

### PR DESCRIPTION
**Pull request by challgren**
Fixes #300

### CODE REVIEW

File: templates/bake/element/form.twig [hunk 1]
```diff
@@ -23,6 +24,7 @@
                 {{- "\n" }}
             {%- endif %}
         {%- endif %}
+        {% endif %}
```
Comment (ADmad) [inline templates/bake/element/form.twig]: This should probably be `{%- endif %}` to keep the whitespacing as before.

Comment (challgren) [reply templates/bake/element/form.twig]: Done thanks! I'm pretty new to twig

**Comment by ADmad** [comment]
Please rebase your bugfix branch onto upstream `cake-4-bs-4`. I have pushed new commits to fixed other unrelated testsuite failures.

File: templates/bake/element/form.twig [hunk 2]
```diff
@@ -3,7 +3,8 @@
     <fieldset>
         <legend><?= __('{{ action|humanize }} {{ singularHumanName }}') ?></legend>
         <?php
-{% for field in fields if field not in primaryKey %}
+{% for field in fields %}
+{%- if field not in primaryKey %}
```
Comment (ADmad) [inline templates/bake/element/form.twig]: I don't think this extra `if` is necessary. Could just include the additional check in `if` statement below:

`{%- if keyFields[field] and field not in primaryKey %}`

Comment (challgren) [reply templates/bake/element/form.twig]: Yeah thats above my pay grade for twig knowledge. I did that and now its adding a control for id thats not expected by the tests

Comment (ADmad) [reply templates/bake/element/form.twig]: Okay, I'll try to optimize it post merge.

Comment (challgren) [reply templates/bake/element/form.twig]: Ok yeah we want the extra if, combining them will add the primary key

**Pull request merged**